test/tbb/test_join_node.h

```diff
@@ -26,6 +26,12 @@
 #endif
 #endif
 
+#ifdef __MIPS
+    #define MAX_TUPLE_TEST_SIZE_LOCAL 3
+#else
+    #define MAX_TUPLE_TEST_SIZE_LOCAL MAX_TUPLE_TEST_SIZE
+#endif
```

Comment:
I would propose to redefine the `MAX_TUPLE_TEST_SIZE` instead of introducing new macro?
```suggestion
#if TBB_TEST_LOW_WORKLOAD && defined(MAX_TUPLE_TEST_SIZE)
    #undef MAX_TUPLE_TEST_SIZE
    #define MAX_TUPLE_TEST_SIZE 3
#endif
```